docs/dev/run_the_etl.rst

```diff
@@ -14,10 +14,27 @@ These instructions assume you have already gone through the :ref:`dev_setup`.
 Database initialization
 -----------------------
 
-Before we run anything (and after we make changes to the database schema), we'll
-need to make sure that the schema in the database actually matches the schema
-in the code - run ``pudl_reset_db`` to delete whatever is already there and
-recreate the database with the right schema.
+Before we run anything, we'll need to make sure that the schema in the database
+actually matches the schema in the code - run ``alembic upgrade head`` to create
+the database with the right schema.
```

Comment:
If folks have an existing pudl.sqlite file but haven't used alembic they'll need to delete it before running `alembic upgrade head` right?

Reply:
Should we include this in the docs? Something like: "If you have an existing pudl.sqlite that wasn't setup up with alembic, you'll need to delete the database and run `alembic upgrade head`".

Reply:
As an alembic novice I say yes, handholding in the transition is useful!

Reply:
Yeah, just added a reference!